addons/web_editor/models/ir_ui_view.py

```diff
@@ -6,7 +6,8 @@
 from lxml import etree, html
 
 from odoo.exceptions import AccessError
-from odoo import api, fields, models
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
```

Comment:
here it's slightly better to have both import at same like (`import AccessError, ValidationError`) or possibly you could do that and fix the order putting `from odoo` above `from odoo.exceptions`)